neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/dialect/impl/EntityKeyMetadataWithDiscriminator.java

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
In the commit message:
s/this queries needs now/these queries now need/
s/therefor/therefore/